x-pack/legacy/plugins/siem/public/components/open_timeline/timelines_table/actions_columns.tsx

```diff
@@ -26,7 +32,12 @@ export const getActionsColumns = ({
   showDeleteAction: boolean;
 }) => {
   const deleteTimelineColumn = {
-    align: 'center',
+    align: 'right',
+    name: (
```

Comment:
@andrew-goldstein does this look ok to you? it adds a header icon to these columns

Reply:
Would you be willing to check with @MichaelMarcialis to see how feels about this change, since it's a departure from the wireframe shown in the screenshot below?

![timelines-page](https://user-images.githubusercontent.com/4459398/61677415-caeefd80-acbc-11e9-927c-b40821da835e.png)

Reply:
Ah, missed my window to respond. I'll have a look at the change when I pull in `master` on my local.